.github/workflows/enforce-issue-template.yml

```diff
@@ -0,0 +1,35 @@
+name: enforce issue template
```

Comment:
```suggestion
name: Enforce issue template
```

Shall we say "validate" to soften the blow? 😀

Reply:
That's friendlier for sure. :)

P.S. Lowercase is the precedent:

<img width="299" alt="Screen Shot 2021-01-14 at 10 26 42 AM" src="https://user-images.githubusercontent.com/134455/104611339-015bcd80-5653-11eb-899c-0d443784e812.png">

Reply:
>P.S. Lowercase is the precedent

That's the wrong precedent tho. We started using upper-case at the start at least as it looks pretty bad on action listing page. (See all new `Release` actions)

Reply:
https://github.com/getsentry/onpremise/pull/814